doc/contribute.md

```diff
@@ -174,7 +174,8 @@ By making a contribution to this project, I certify that:
 All commit messages must contain the `Signed-off-by` line with an email address
 that matches the commit author. This line asserts the Developer Certificate of Origin.
 
-When committing, use the `--signoff` (or `-s`) flag:
+[Set up a prepare-commit-msg git hook](#do-i-really-have-to-add-the--s-flag-to-each-commit),
```

Comment:
Perhaps we could make this suggestion more of an after-thought in a separate paragraph at the end of this section?

Reply:
Good suggestion, done!